indico/modules/events/registration/templates/management/regform_list.html

```diff
@@ -13,9 +13,9 @@
 
 {% block content %}
     {% if not event.has_feature('payment') %}
-        <div id="payment-disabled-notice" class="action-box highlight">
-            <div class="section">
-                {% block payments_disabled %}
+        {% block payments_disabled %}
```

Comment:
Again, I would consider adding `_box` to the block name: 

```suggestion
        {% block payments_disabled_box %}
```